src/lib/test/RawText.svelte

```
@@ -0,0 +1,11 @@
+<script lang="ts">
+    interface Props {
+        text?: string
+    }
+    const { text }: Props = $props()
+
+    $inspect('RawText.svelte', text)
+</script>
+
+<!-- trunk-ignore(eslint/svelte/no-at-html-tags) -->
+{@html text}
```

ESLint

[error] `{@html}` can lead to XSS attack.

(svelte/no-at-html-tags)